db/fixtures/development/00_seeds.rb

```diff
@@ -153,6 +177,10 @@
 
   {id: 14, date: "2022-03-11", start_time: "12:00", end_time: "19:00", conference_id: 5, internal: false},
   {id: 15, date: "2022-02-16", start_time: "19:00", end_time: "21:00", conference_id: 5, internal: true}, #Pre event
+
+  # CNSEC2022
+  {id: 16, date: "2022-08-05", start_time: "12:00", end_time: "19:00", conference_id: 6, internal: false},
```

Comment:
```suggestion
  {id: 16, date: "2022-08-05", start_time: "13:00", end_time: "19:00", conference_id: 6, internal: false},
```

13時スタートらしいので
https://github.com/cloudnativedaysjp/cnsec2022/blob/main/cnsec2022_sponsor_guide(JP).md